includes/polling/loadbalancers/f5-ltm.inc.php

```diff
@@ -180,8 +190,13 @@
             d_echo("Type is unknown: ".$type."\n");
             continue;
         }
+        if ($type == 'f5-ltm-poolmember') {
+            $nodename = $f5_stats['ltmPoolMemberStatEntryName']['1.3.6.1.4.1.3375.2.2.5.4.3.1.28.'.$UID];
+            $tags = compact('rrd_name', 'rrd_def', 'type', 'hash', 'label', 'nodename');
```

Comment:
This RRD file will already exist so you can't update it with an additional piece of data. You would need to create a second RRD to store this information in.

Also, you aren't outputting this anywhere so why store it in the first place?